lib/logstash/filters/grok.rb

```diff
@@ -223,7 +223,7 @@ class LogStash::Filters::Grok < LogStash::Filters::Base
     config :overwrite, :validate => :array, :default => []
```

Comment:
This file seems to only have whitespace and logging changes and does not appear related to the description in the PR? Can we do without it?